plugin/storage/integration/integration.go

```diff
@@ -532,7 +532,7 @@ func (s *StorageIntegration) RunAll(t *testing.T) {
 	t.Run("ArchiveTrace", s.testArchiveTrace)
 	t.Run("GetOperations", s.testGetOperations)
 	t.Run("GetTrace", s.testGetTrace)
-	t.Run("GetLargeSpans", s.testGetLargeSpan)
+	t.Run("GetLargeSpans", s.TestGetLargeSpan)
```

Comment:
the intention is that the other tests would call RunAll, not cherry-pick specific tests